src/librustc_codegen_utils/target_features.rs

```diff
@@ -0,0 +1,113 @@
+// WARNING: the features after applying `to_llvm_feature` must be known
```

Comment:
So you moved these so you could access them in your backend?

Reply:
Yep. This commit wasn't supposed to be part of this PR (the change has already been merged). See my comment above. Won't happen again!